content/docs/guides/crosswalk/aws/api-gateway.md

```diff
@@ -520,27 +520,27 @@ https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-method-
 
 ## Controlling and Managing Access to APIs
 
-API Gateway supports several mechanisms for controlling and managing access to your APIs. This includes authentication
+AWS API Gateway supports several mechanisms for controlling and managing access to your APIs. This includes authentication
 and authorization -- e.g., resource policies, standard AWS IAM roles and policies, Cognito user pools, and Lambda
 authorizers -- other access control tasks -- e.g., cross-origin resource sharing (CORS), client-side SSL certificates,
 and Amazon Web Application Firewall (WAF) -- and limiting access to authorized clients through usage plans and API keys.
 
 The `awsx.apigateway.API` class supports three specific methods of controlling access to your APIs:
 
-* *Amazon Cognito user pools* let you create customizable authentication and authorization for your APIs.
+* *AWS Cognito user pools* let you create customizable authentication and authorization for your APIs.
```

Comment:
I believe for this one, the product name actually is Amazon Cognito, not AWS Cognito.
```suggestion
* *Amazon Cognito user pools* let you create customizable authentication and authorization for your APIs.
```